ldcbench.http-node/src/main/java/org/dice_research/ldcbench/nodes/http/simple/dump/DumpFileResource.java

```
@@ -59,7 +60,11 @@ public static DumpFileResource create(int domainId, String[] resourceUriTemplate
             LOGGER.error("Couldn't create dump file.", e);
         } catch (ReflectiveOperationException e) {
             LOGGER.error("Couldn't create dump file.", e);
-        }
+        }catch (ParserException e) {
+			LOGGER.error("Couldn't create dump file.", e);
+		} catch (NotFoundException e) {
+			LOGGER.error("Couldn't create dump file.", e);
+		}
```

There are a lot of catch clauses. It does not make much sense to have so many if all exceptions are handled in the same way.
Either you list all of them in one block:
```
catch (IOException | NoSuchMethodException | ... e) { ...
```
or you simply catch all exceptions:
```
catch (Exception e) { ...
```
I would suggest the latter :thinking: